### PR TITLE
[WIP] Dynamic graphs

### PR DESCRIPTION
**Pull request by CDuPlooy**
On mobile.

#### Screenshot
screen shot 2018-10-08 at 11 07 21

### CODE REVIEW

File: src/app/pages/services/service-detail-view/service-detail-view.component.ts [hunk 1]
```diff
@@ -87,15 +87,20 @@ export class ServiceDetailViewComponent implements OnInit {
...
     } else {
       this.port = '-';
     }
   }
 
   getImage() {
-    this.image = this.serviceModel.Spec.TaskTemplate.ContainerSpec.Image.split('@')[0];
+    this.image = this.serviceModel.Spec.TaskTemplate.ContainerSpec.Image.split(
+      '@'
+    )[0];
```
Comment (egeldenhuys) [inline src/app/pages/services/service-detail-view/service-detail-view.component.ts]: Did `prettier` change this formatting?

**Review by egeldenhuys — CHANGES_REQUESTED** [review]
The Offline/Online nodes chart is fine :+1: , however the chart on the left is comparing values that cannot be compared. The number of nodes compared to the number of networks does not provide any insights except "I have 1 node and 35 networks". Can we remove this chart or replace it with boxes showing the number of Nodes/networks/etc ?

The colour scheme is also better :+1:

**Comment by CDuPlooy** [comment]
I have absolutely no idea. X)

As per the chart; I'll make those changes when I have time :)

**Comment by annamarieHelberg** [comment]
I think taking a look at swarmpit and portainer's graphs would help. But thank you for implementing it :+1:

**Review by egeldenhuys — CHANGES_REQUESTED** [review]
I realise logic tends to leave us at such late times, but can you please give 8a232da2233425481d84eed7c5096ac69571a462 a relevant comment?

**Comment by CDuPlooy** [comment]
> I think taking a look at swarmpit and portainer's graphs would help. But thank you for implementing it

I something like [this](https://github.com/swarmstack/swarmstack) would be nice.

I've removed the redundant graph and added cards.

File: src/app/pages/home/home.component.ts [hunk 2]
```diff
@@ -73,7 +73,7 @@ export class HomeComponent implements OnInit, OnDestroy {
     this.updateChartA();
     this.updateChartB();
     this.count_types();
-    this.page_start = interval(5000).subscribe(() => {
+    this.page_start = interval(30000).subscribe(() => {
```
Comment (egeldenhuys) [inline src/app/pages/home/home.component.ts:76]: This should help

**Review by egeldenhuys — APPROVED** [review]
Looks good

**Review by ghost — APPROVED** [review]
I like the new layout looks good !

**Pull request merged**